### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by dobrzyckahanna**
Upgraded Bootstrap, nav-bar and modal added.

### CODE REVIEW

**Comment by dobrzyckahanna** [comment]
First pull request, ist it succesfull?

File: templates/validator/projects.html [hunk 1]
```diff
@@ -1,33 +1,66 @@
...
+<div class="container">
+    <div class="modal fade" id="newProject">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h4 class="modal-title">Add new project</h4>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="close">&times;</button>
+                </div>
+                <div class="modal-body">
+                    <form action="/action_page.php">
```
Comment (misiekofski) [inline templates/validator/projects.html]: this one will need to be changed to "/projects" not action_page.php
also method="POST" is not here

File: templates/validator/projects.html [hunk 2]
```diff
@@ -1,33 +1,66 @@
...
+                        Name:<br>
+                        <input type="text" name="projectName" id="projName"><br>
```
Comment (misiekofski) [inline templates/validator/projects.html]: why do we use different name and id?

File: templates/validator/projects.html [hunk 3]
```diff
@@ -1,33 +1,66 @@
...
+                        Description:<br>
+                        <textarea rows="3" cols="50" name="projectDescription" id="projDesc"></textarea><br>
```
Comment (misiekofski) [inline templates/validator/projects.html]: why do we use different name and id?

File: templates/validator/projects.html [hunk 4]
```diff
@@ -1,33 +1,66 @@
...
+                    </form>
+                </div>
+                <div class="modal-footer">
+                    <form>
+                        <input type="submit" class="btn btn-primary" name="submitButton" id="buttonAddNewProject"
```
Comment (misiekofski) [inline templates/validator/projects.html]: why input button is in new form? that might not work

File: templates/validator/stories.html [hunk 5]
```diff
@@ -3,42 +3,71 @@
...
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h4 class="modal-title">Add new story</h4>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="close">&times;</button>
+                </div>
+                <div class="modal-body">
+                    <form action="/action_page.php">
+                        Name:<br>
+                        <input type="text" name="projectName" id="projName"><br>
```
Comment (misiekofski) [inline templates/validator/stories.html]: why different id and name?

File: templates/validator/stories.html [hunk 6]
```diff
@@ -3,42 +3,71 @@
...
+                        Description:<br>
+                        <textarea rows="3" cols="50" name="projectDescription" id="projDesc"></textarea><br>
```
Comment (misiekofski) [inline templates/validator/stories.html]: why different id and name?

File: templates/validator/stories.html [hunk 7]
```diff
@@ -3,42 +3,71 @@
...
+<div class="container">
+    <div class="modal fade" id="newStory">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h4 class="modal-title">Add new story</h4>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="close">&times;</button>
+                </div>
+                <div class="modal-body">
+                    <form action="/action_page.php">
```
Comment (misiekofski) [inline templates/validator/stories.html]: action="/stories" method="POST"

File: templates/validator/stories.html [hunk 8]
```diff
@@ -3,42 +3,71 @@
...
+                        Name:<br>
+                        <input type="text" name="projectName" id="projName"><br>
+                        Description:<br>
+                        <textarea rows="3" cols="50" name="projectDescription" id="projDesc"></textarea><br>
+                    </form>
+                </div>
+                <div class="modal-footer">
+                    <form>
```
Comment (misiekofski) [inline templates/validator/stories.html]: another form with input button?

File: templates/validator/stories.html [hunk 9]
```diff
@@ -3,42 +3,71 @@
...
+                    <button type="button" class="close" data-dismiss="modal" aria-label="close">&times;</button>
+                </div>
+                <div class="modal-body">
+                    <form action="/action_page.php">
+                        Name:<br>
+                        <input type="text" name="projectName" id="projName"><br>
+                        Description:<br>
+                        <textarea rows="3" cols="50" name="projectDescription" id="projDesc"></textarea><br>
+                    </form>
+                </div>
```
Comment (misiekofski) [inline templates/validator/stories.html]: missing dropdown with statuses, the same as in projects

File: templates/validator/add_project_form.html [hunk 10]
```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
```
Comment (misiekofski) [inline templates/validator/add_project_form.html]: what is this file? why do we need it here?

**Pull request merged**